streaming/src/main/scala/org/apache/spark/streaming/scheduler/rate/RateEstimator.scala

```diff
@@ -61,7 +60,8 @@ object RateEstimator {
         val proportional = conf.getDouble("spark.streaming.backpressure.pid.proportional", 1.0)
         val integral = conf.getDouble("spark.streaming.backpressure.pid.integral", 0.2)
         val derived = conf.getDouble("spark.streaming.backpressure.pid.derived", 0.0)
-        new PIDRateEstimator(batchInterval.milliseconds, proportional, integral, derived)
+        val minRate = conf.getDouble("spark.streaming.backpressure.pid.minRate", 100)
```

Comment:
Yeah i thought so. for now this is an intrinsic characteristic of the pid algorithm, so we can it like this for now. I am not making these parameters public in this release, so we can figure this out later.